fps/go.mod

```diff
@@ -0,0 +1,25 @@
+module fps
```

Comment:
if I put `fortio.org/terminal/fps` here to make go install work I get
```
go: fortio.org/terminal/fps: ambiguous import: found package fortio.org/terminal/fps in multiple modules:
	fortio.org/terminal v0.27.3-0.20250407185218-eb480fe9cdf9 (/Users/dl/go/pkg/mod/fortio.org/terminal@v0.27.3-0.20250407185218-eb480fe9cdf9/fps)
	fortio.org/terminal/fps (/Users/dl/dev/fortio.org/terminal/fps)
```

Reply:
You need to do this process with two commits.

1. to create the module
2. second to update the dependency at the top level

Reply:
Thanks! that's what I thought I did do (there are 2 commits) but I will try again

also found https://github.com/go-modules-by-example/index/blob/master/009_submodules/README.md while doing archeology/searches (and yet it's not (yet) working for this)

Reply:
oh you mean it's the parent that complains it's responsible because it doesn't see the child? let's see...3 commits then

Reply:
well go mod tidy is now happy but
```
go: fortio.org/terminal/fps@47adc0c2aed22c11c65390c477c309b578ce2be9: module fortio.org/terminal@47adc0c2aed22c11c65390c477c309b578ce2be9 found (v0.27.3-0.20250407202059-47adc0c2aed2), but does not contain package fortio.org/terminal/fps
```
maybe I need an extra meta tag... not sure

Reply:
that was it